docs(storybook): docs cleanup (#6495)
adobe/spectrum-web-components: 2nd-gen/packages/swc/.storybook/resources/build-with-ai.mdx

Reviewer: ```suggestion
This page covers resources to help AI coding tools understand and use Spectrum Web Components.
```

Reviewer: ```suggestion
This skill is activated when your project uses `@spectrum-web-components/*` packages or `sp-*` custom elements.
```

Reviewer: ```suggestion
It includes guides, component API references, accessibility guidance, and integration examples.
```

Reviewer: ```suggestion
This skill is activated when migrating `sp-*` elements to `swc-*` elements from `@adobe/spectrum-wc`.
```

Reviewer: ```suggestion
It includes the full coexistence walkthrough, migration guides for each component, and a CSS property
```

[Agent Skills](https://agentskills.io) are bundles of instructions and reference files that
your AI coding tool can load when they are relevant to a task.

> Two skills are available for Spectrum Web Components. There is not yet a dedicated skill for
> building a new Gen2 project with no Gen1 code to migrate; in the meantime, point your AI client
> at this documentation site directly for Gen2 component and pattern references.

Reviewer: ```suggestion
2 skills are available for Spectrum Web Components. There is not yet a dedicated skill for
building new, Gen2-only projects. In the meantime, point your AI client
to this documentation site directly for Gen2 component and pattern references.
```

## llms.txt

An [`llms.txt`](https://opensource.adobe.com/spectrum-web-components/llms.txt) file is available
for the **Gen1** documentation site only. AI tools that support the
[llms.txt convention](https://llmstxt.org) can use this file to discover and load the right

> Gen1 documentation page for a given task. A Gen2-specific `llms.txt` is not yet available;

Reviewer: ```suggestion
Gen1 documentation page for a given task. A Gen2-specific `llms.txt` is not yet available.
```

> until then, use the Agent Skills above for Gen2 guidance.

Reviewer: ```suggestion
Until then, use the Agent Skills above for Gen2 guidance.
```
